File: How many even numbers/howmanyevens.java
import java.util.Scanner;

public class howmanyevens{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int size = Integer.parseInt(scanner.nextLine());
        String[] sarr = scanner.nextLine().trim().split(" ");

        int total = 0;
        for (String s : sarr) {
            int num = Integer.parseInt(s);
            if ( num %2 == 0){
                total++;
            }
        }

        System.out.println(total);

        scanner.close();
    }
}

File: How many even numbers/howmanyevens.cpp
#include <iostream>

using namespace std;

int main(){

    int size;

    cin>>size;
    int total;

    for(int i=0;i<size; i++){
        int num;
        cin>>num;
        if(num%2==0){
            total++;
        }
    }

    cout<<total<<endl;

    return 0;
}